cli/src/package.rs

```diff
@@ -92,7 +125,23 @@ impl PackageCommand {
                     ..config
                 };
 
-                manifest.snapshot_dependencies(config)?;
+                manifest.snapshot_dependencies(config.clone())?;
```

Comment:
Out of curiosity, why do we need to take a config as an owned value?

Reply:
Hm, I think maybe I can get rid of this one. In some other places it's just convenient to store copies of the config so that we can figure out where various files go.

Reply:
Indeed, we now no longer store `Config` in `Snapshot`.